.github/workflows/checkAndSubmitAddonMetadata.yml

```diff
@@ -179,6 +179,137 @@ jobs:
       uses: peter-evans/close-issue@v3
       with:
         issue-number: ${{ inputs.issueNumber }}
+  createReviewComment:
+    # jq for windows has issues parsing multiline strings (e.g. CRLF),
+    # use linux instead.
+    runs-on: ubuntu-latest
+    needs: [checkMetadata]
+    strategy:
+      matrix:
+        python-version: [ 3.11 ]
+    permissions:
+      contents: write
+      discussions: write
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      ref: ${{ inputs.headRef }}
+    - name: Get add-on name and version
+      id: getAddonNameAndVersion
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const addonFilename = ${{ needs.getAddonFilename.outputs.addonFileName }}
+          const addonName = addonFilename.split('/')[1]
```

Comment:
this is still just using the add-on file name. Please get the add-on name from the meta data

Reply:
this is still unresolved